doc/HttpLuaModule.wiki

```diff
@@ -2088,23 +2088,40 @@ These constants are usually used in [[#ngx.location.capture|ngx.location.capture
 '''context:''' ''init_by_lua*, set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, body_filter_by_lua, log_by_lua*, ngx.timer.*''
 
 <geshi lang="nginx">
+  value = ngx.HTTP_CONTINUE (100) (first added in the v0.9.16 release)
+  value = ngx.HTTP_SWITCHING_PROTOCOLS (102) (first added in the v0.9.16 release)
   value = ngx.HTTP_OK (200)
   value = ngx.HTTP_CREATED (201)
+  value = ngx.HTTP_ACCEPTED (202) (first added in the v0.9.16 release)
+  value = ngx.HTTP_NO_CONTENT (204) (first added in the v0.9.16 release)
+  value = ngx.HTTP_PARTIAL_CONTENT (206) (first added in the v0.9.16 release)
   value = ngx.HTTP_SPECIAL_RESPONSE (300)
   value = ngx.HTTP_MOVED_PERMANENTLY (301)
   value = ngx.HTTP_MOVED_TEMPORARILY (302)
   value = ngx.HTTP_SEE_OTHER (303)
   value = ngx.HTTP_NOT_MODIFIED (304)
+  value = ngx.HTTP_TEMPORARY_REDIRECT (307) (first added in the v0.9.16 release)
   value = ngx.HTTP_BAD_REQUEST (400)
   value = ngx.HTTP_UNAUTHORIZED (401)
+  value = ngx.HTTP_PAYMENT_REQUIRED (402) (first added in the v0.9.16 release)
   value = ngx.HTTP_FORBIDDEN (403)
   value = ngx.HTTP_NOT_FOUND (404)
   value = ngx.HTTP_NOT_ALLOWED (405)
+  value = ngx.HTTP_NOT_ACCEPTABLE (406) (first added in the v0.9.16 release)
+  value = ngx.HTTP_REQUEST_TIMEOUT (408) (first added in the v0.9.16 release)
+  value = ngx.HTTP_CONFLICT (409) (first added in the v0.9.16 release)
   value = ngx.HTTP_GONE (410)
+  value = ngx.HTTP_UPGRADE_REQUIRED (426) (first added in the v0.9.16 release)
+  value = ngx.HTTP_TOO_MANY_REQUESTS (429) (first added in the v0.9.16 release)
+  value = ngx.HTTP_CLOSE (444) (first added in the v0.9.16 release)
+  value = ngx.HTTP_ILLEGAL (451) (first added in the v0.9.16 release)
```

Comment:
This status code for "Unavailable For Legal Reasons" is not yet universally adopted and at least Microsoft has its own definition, "Redirect".